includes/customers/customer-actions.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Customer Actions.
+ *
+ * Hooks that are triggered when customer-based actions occur.
+ *
+ * @package     EDD
+ * @subpackage  Actions
+ * @copyright   Copyright (c) 2020, Sandhills Development, LLC
+ * @license     http://opensource.org/licenses/gpl-2.0.php GNU Public License
+ * @since       3.0
+ */
+
+/**
+ * When a new primary address is added to the database, any other primary addresses should be demoted.
+ *
+ * @param string $old_value The previous value of `is_primary`.
+ * @param string $new_value The new value of `is_primary`.
+ * @param int $item_id      The address ID in the edd_customer_addresses table.
+ * @return void
+ */
+function edd_demote_customer_primary_addresses( $old_value, $new_value, $item_id ) {
+	if ( ! $new_value ) {
+		return;
+	}
+	$customer_addresses         = new EDD\Database\Queries\Customer_Address();
+	$address                    = $customer_addresses->get_item( $item_id );
+	$previous_primary_addresses = $customer_addresses->query(
```

Comment:
Similar to [Ashley's other suggestion](https://github.com/easydigitaldownloads/easy-digital-downloads/pull/8146/files#discussion_r516671612), could these use `edd_get_customer_addresses()` and `edd_fetch_customer_address()`?